client/apps/edit/components/admin/components/autocomplete_list_metaphysics.jsx

```diff
@@ -38,7 +38,7 @@ export class AutocompleteListMetaphysics extends Component {
         return Queries.PartnersQuery
       }
       case 'partner_shows': {
-        return Queries.PartnersQuery
+        return Queries.ShowsQuery
```

Comment:
In addition to this, there's an error with this query because MP does not yet support `ids` as an argument. I feel like you talked about this before so lmk if you already have a working PR otherwise I'm happy to implement in MP

Reply:
Hm maybe I missed shows -- can look at implementing tomorrow